NEWS.md

```diff
@@ -58,8 +58,10 @@
   runtime. Use the new `--qemu-gui` option to start qemu in its graphical interface.
 - Removed `--netdev`. Can be replaced by manually installing systemd-networkd, putting a network file in the
   image and enabling systemd-networkd.
-- If `mkosi.extra/` or `mkosi.skeleton/` exist, they are now always used instead of only when no explicit
-  extra/skeleton trees are defined.
+- If `mkosi.extra/` exists, it is now always used instead of only when no explicit extra trees are defined.
+- Removed `--qcow2` option in favor of supporting only raw disk images as the disk image output format.
+- Removed `--bmap` option as it can be trivially added manually by utilizing a finalize script.
```

Comment:
That's maybe very nitpicky, but the commit message dropping `--bmap` too only mentions `--qcow2`, adding `--bmap` there might be nice when looking for it in the log.